README.md

```diff
@@ -1 +1,164 @@
-# Ruby-Capstone
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 Catalog of my things](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# [📖 Catalog of my things] <a name="about-project"></a>
+In this project, We built a console app that stores books, label, music albums, games and their authors. In this initial step, We have implemented the classes to represent Books, Music Albums and Games with other associate classes like authors, genre and label.
+[📖 Catalog of my things] use functionality of classes and objects in Ruby, Implement encapsulation and inheritance with Ruby and Run a program using the command line.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- Ruby
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- Add a Book and associated label👌
+- Add a music album with genre 💯
+- Add a Games with authors👌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://drive.google.com/file/d/1ONBwtug8HgDCt2sMT0WAMFIaIS594XkS/view?usp=sharing)
+
```

Comment:
- [ ] Great job recording a video, but we have to follow some instructions for the video, such as the length, a piece of code you are proud of, etc. I have highlighted the requirements to follow for the video in the image below. 

![video](https://github.com/Newtayo/Ruby-Capstone/assets/53124548/5bd99bd8-052b-473e-af9c-928d3c1d0b41)